ValidateCommitMessage.yml

```
@@ -33,9 +33,66 @@ jobs:
+      rm -rf PythonRestAPI
+    displayName: 'Test PostgreSQL DB API creation'
+
+  - script: |
+      python -m venv venv
+      source venv/bin/activate
+      pip install --upgrade pip
+      pip install -r requirements.txt
+      pwsh -Command "& { ./tests/Databases/SQLServer/test_sqlserver.ps1 }"
+      rm -rf PythonRestAPI
+    displayName: 'Test SQL Server DB API creation'
+    
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)